app/views/publications/v2/monthly_statistics/show.html.erb

```diff
@@ -0,0 +1,247 @@
+      <dl class="gem-c-metadata">
+        <dt class="gem-c-metadata__term">Published</dt>
+        <dd class="gem-c-metadata__definition"><%= @presenter.publication_date.to_fs(:govuk_date) %></dd>
+
+        <% if @presenter.current_cycle? %>
+          <dt class="gem-c-metadata__term">Next update</dt>
+          <dd class="gem-c-metadata__definition"><%= @presenter.next_publication_date.to_fs(:govuk_date) %></dd>
+        <% end %>
+      </dl>
+    </div>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-body govuk-!-font-size-16 govuk-!-margin-bottom-1 govuk-!-margin-top-5">Contents</h2>
+
+    <ol class="govuk-list govuk-!-margin-bottom-8 app-toc">
+      <li>
+        <a href="#introduction" class="govuk-link app-toc__link">
+          <span class="app-toc__number">1.</span>
+          <span class="app-toc__content">Introduction</span>
+        </a>
+      </li>
+      <li>
+        <a href="#headline-statistics" class="govuk-link app-toc__link">
+          <span class="app-toc__number">2.</span>
+          <span class="app-toc__content">Candidate headline statistics</span>
+        </a>
+      </li>
+      <li>
+       <a href="#by-age-group" class="govuk-link app-toc__link">
+         <span class="app-toc__number">3.</span>
+         <span class="app-toc__content">Candidate age group</span>
+       </a>
+     </li>
+      <li>
+        <a href="#applications-by-sex" class="govuk-link app-toc__link">
+          <span class="app-toc__number">4.</span>
+          <span class="app-toc__content">Candidate sex</span>
+        </a>
+      </li>
+      <li>
+        <a href="#applications-by-area" class="govuk-link app-toc__link">
+          <span class="app-toc__number">5.</span>
+          <span class="app-toc__content">Candidate area</span>
+        </a>
+      </li>
+      <li>
+        <a href="#applications-by-course-phase" class="govuk-link app-toc__link">
+          <span class="app-toc__number">6.</span>
+          <span class="app-toc__content">Course phase</span>
+        </a>
+      </li>
+      <li>
+        <a href="#downloads" class="govuk-link app-toc__link">
+          <span class="app-toc__number">7.</span>
+          <span class="app-toc__content">Download the data</span>
+        </a>
+      </li>
+      <li>
+        <a href="#footnotes" class="govuk-link app-toc__link">
+          <span class="app-toc__number">8.</span>
+          <span class="app-toc__content">Footnotes</span>
+        </a>
+      </li>
+    </ol>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-l" id="introduction">1. Introduction</h2>
+
+    <p class="govuk-body">These statistics cover applications for courses in England starting in the <%= @presenter.next_cycle_name %> academic year. To allow for comparison statistics covering the <%= @presenter.current_cycle_name %> academic year are also included.</p>
```

Comment:
This took me a second to remember we differentiate between cycle year and academic year. I wonder if something like `presenter.upcoming_academic_cycle_year` (I'm sure someone can think of a better name than me) might help others who don't have the context. What are your views?

Reply:
Yes, the mixture of academic year and recruitment cycle is really confusing.